http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/AppV2Routes.scala

```diff
@@ -169,13 +169,15 @@ object AppV2Routes {
       for {
         c <- x.downField("kubernetesRuntimeConfig").as[Option[KubernetesRuntimeConfig]]
         a <- x.downField("appType").as[Option[AppType]]
+        s <- x.downField("accessScope").as[Option[AppAccessScope]]
         d <- x.downField("diskConfig").as[Option[PersistentDiskRequest]]
         l <- x.downField("labels").as[Option[LabelMap]]
         cv <- x.downField("customEnvironmentVariables").as[Option[LabelMap]]
         dp <- x.downField("descriptorPath").as[Option[Uri]]
         ea <- x.downField("extraArgs").as[Option[List[String]]]
       } yield CreateAppRequest(c,
                                a.getOrElse(AppType.Galaxy),
+                               s,
```

Comment:
Not enforcing the default here as I need to do it anyways in `AppSamResourceId` for the V1 apps